website/content/maintainers/github/templates/recommended/reviewing.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "HowTo: Make a Reviewing Guide"
+linkTitle: "Reviewing Guide"
+date: 2021-09-20
+---
+
+| Audience of this HowTo | Audience of REVIEWING.md | Required by CNCF    |
+| ---------------------- | ------------------------ | ------------------- |
+| Maintainers            | Reviewers                | No, but encouraged! |
+
+
+This HowTo is for project maintainers to help guide them in making intentional decisions about how they want to run their review process, and what they want to require of reviewers. A written review process can ensure that reviews are consistent and adhere to your project values. Reviewers default to patterns set from their own experiences in open source, or processes used at previous jobs. Don't assume that whatever review process you have experienced is universal.
+
+The goal of a REVIEWING.md file is to help set expectations and guardrails for reviewers. It should cover reviewer responsibilities, such as following a common checklist for all reviews, requiring that comments are constructive and kind, or applying labels to the pull request. It can also help a reviewer set boundaries, such as reasonable turnaround times on feedback or how much assistance they are expected to provide new contributors. 
+
+The benefits of defining your review process include:
+* Reduce contributor churn.
+* Improve likelihood of new contributors becoming regular contributors.
+* Reduces misunderstandings and confusion during reviews.
+
+## Fill out the template
+
+The [REVIEWING.md template](https://github.com/cncf/project-template/blob/main/REVIEWING.md) is located in the CNCF [project-template repository](https://github.com/cncf/project-template).
+
+Copy the template file into your repository.
+There are instructions for filling out the template that look like the example below:
+
+![screenshot of the REVIEWING.md template, there is a link to instructions, and a warning emoji with text explaining how to fill out this section of the template](/maintainers/github/templates/sample-instructions.png)
+
+When you finish editing the template, remove the Instruction links that explain how to fill out the template. Also remove any ⚠️ prompts and their explanatory text.
+
+Below are instructions for each section in the template:
+
+### The Reviewer Role
+
+Define who is allowed to review pull requests and what rights reviewers have.
+
+* Must all reviewers have write access to the repository?
+* Are maintainers and reviewers separate roles? 
+* Is everyone encouraged to review code, even if they aren't officially reviewers?
+
+### Values
+
+We encourage you to define the values that reviewers should model. A review is a new contributor's first interaction with a project, often with the community following along. Some suggestions are included in the template and you should include any additional project values as needed.
+
+### Process
```

Comment:
I would also suggest adding in space for any automated checks and tooling such as linters so Reviewers know their obligations to review before/after the linter performs, and that contributors should/don't need to resolve any errors from those checks prior to expecting a review to occur.